src/netex_idf/lines.rs

```diff
@@ -273,20 +272,30 @@ mod tests {
                      <Name>TRANSDEV IDF RAMBOUILLET 2</Name>
                   </Operator>
                </organisations>
-            </ResourceFrame>
+            </ResourceFrame>"#;
+        let resource_frame_organisations_1: Element = xml.parse().unwrap();
+        let xml = r#"
             <ResourceFrame version="any" id="STIF:CODIFLIGNE:ResourceFrame:2">
                <organisations>
                   <Operator version="any" id="STIF:CODIFLIGNE:Operator:015">
                      <Name>TRANSDEV IDF RAMBOUILLET 3</Name>
                   </Operator>
                </organisations>
-            </ResourceFrame>
-         </frames>
-      </CompositeFrame>
-   </dataObjects>
-</root>"#;
-        let root: Element = xml.parse().unwrap();
-        let (networks, companies, _) = make_networks_companies(&root).unwrap();
+            </ResourceFrame>"#;
+        let resource_frame_organisations_2: Element = xml.parse().unwrap();
+        let mut frames = HashMap::new();
+        frames.insert(
+            FrameType::Service,
+            vec![&service_frame_networks_1, &service_frame_networks_2],
+        );
+        frames.insert(
+            FrameType::Resource,
+            vec![
+                &resource_frame_organisations_1,
+                &resource_frame_organisations_2,
+            ],
+        );
+        let (networks, companies, _) = make_networks_companies(&frames).unwrap();
```

Comment:
Unit or functional test here? that is the question to be discussed together.

Reply:
Ok for me at the moment. Anyway these tests will change with the new Netex flows.